ISISComputingGroup/EPICS-ioc
# Ticket 6898 allow 35 danfysiks at once

### Description of work

Added Danfysiks 12 through 35

### To test
Run make iocstartups, then start the ibex server and ibex, check the iocstart tree, the danfysik section should now have 35 entries, make sure they can be started properly.

### Ticket
https://github.com/ISISComputingGroup/IBEX/issues/6898

### Acceptance criteria

35 danfysiks

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
